en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+
+## Step 2: Consume the REST API
+
+You have published the readingList REST API to the Developer Portal where application developers can find it and subscribe their applications to it.
+
+In the previous steps, you played the role of a REST API developer and developed a REST API. In this step, you will play the role of a consumer of this REST API.
+
+To consume the  `readingList` REST API, let's create an application, subscribe it to the REST API, and send a request to it.
+
+### Step 2.1: Create an application
+
+First, let's create an application by following the steps given below:
+
+1. In the top menu of the Developer Portal, click **Applications**.
+
+2. Click **Create**.
+
+3. Enter `readingListApp` as the name of the application and click **Create**.
+
+    Your Application will open on a separate page.
+
+### Step 2.2: Subscribe to your application to the API
+
+To subscribe your application to the API, follow the steps given below:
+
+1. In the left navigation menu, click **Subscriptions**.
+
+2. Click **Add APIs**.
+
+3. Find your REST API and click **Add** for it.
+
+Now your application has subscribed to the `readingList` REST API.
+
+### Step 2.3: Generate a token for your application
+
+To generate a security token that your application can use to invoke the REST API, follow the steps given below:
+
+1. In the left navigation menu, click **OAuth 2.0 Tokens**.
+
+2. Click **Generate Credentials**.
+
+3. Click **Generate Test Token**.
+
+4. Copy the test token that appears by clicking the icon for copying and save it for later use.
+
+5. Expand **Advanced Configurations** and select the **Public Client** checkbox.
+
+6. Click **Update**.
+
+### Step 2.4: Deploy a Web application and invoke the REST API
+
+At present, any user can invoke the `readingList` REST API via the `readingListApp` application (i.e., using its token) and update the same reading list. To allow multiple users to use the application and maintain unique reading lists, you need a front-end application that allows each user to log in with a unique user ID.
+
+In this step, let's deploy a pre-designed front-end application for the REST API, configure Asgardeo as the identity provider, and invoke the REST API with a unique user ID.
+
+
+#### Step 2.4.1: Set up the front-end application
+
+To configure the front-end application by defining the ports it needs to run on, adding the endpoint to the `readingList API`, etc., follow the steps given below:
+
+1. Access Asgardeo at [https://console.asgardeo.io/](https://console.asgardeo.io/) and log in with the same credentials with which you logged in to Choreo.
+
+2. Click **Single-Page Application** to start adding a single-page application.
+
+3. Enter a name for the single-page application and enter `https://localhost:5173` as the authorized redirect URL.
+
+4. Click **Register**.
+
+5. Click **Protocol**.
+
+6. In the **Protocol** tab, make the following changes:
```

Comment:
Shall we put the protocol settings into a table with descriptions instead of step-be-step instructions of granular steps.

Reply:
Did not have a table because the method to set each value is different. Some need to be typed in and then saved by clicking another **Add** button, and some are radio buttons. 

Also, this page is long with many subsections, and thought it might be useful if there are steps mentioning the sections where users can locate each field they need to edit.